### ISSUE TITLE
Check for css animation

### ISSUE THREAD

**Issue by malte-p12** [marionebl/jogwheel#358]
Jogwheel should unplug automatically when there is no css animation definend and plug in automatically when the animation is defined again. This is very important for animations which are defined through media queries.

**Comment by marionebl** [comment]
Good call, will land this with the next feature release, expect it sometime next week
